### PR TITLE
feat: improve audit log

### PR DESCRIPTION
**Pull request by Aleksgav**

### CODE REVIEW

File: object/check.go [hunk 1]
```diff
@@ -293,29 +345,81 @@ func CheckUserPassword(organization string, username string, password string, la
...
+		case ErrorWrongPassword:
+			record := NewRecordBuilder(ctx).WithDetail("Wrong password").Build()
+			SaveOnSuccess(ctx, record)
+		case ErrorLDAPUserNotFound:
+			record := NewRecordBuilder(ctx).WithDetail("LDAP user not found").Build()
+			SaveOnSuccess(ctx, record)
+		case ErrorLDAPError:
+			record := NewRecordBuilder(ctx).WithDetail("LDAP error").Build()
+			SaveOnSuccess(ctx, record)
+		}
```
Comment (dplynskiy) [inline object/check.go:398]: NewRecordBuilder(ctx).WithDetail(err.Error()).Build() isn't enough here instead switch statement?

File: object/user.go [hunk 2]
```diff
@@ -603,10 +603,10 @@ func updateUser(id string, user *User, columns []string) (int64, error) {
 		return 0, err
 	}
 
-	hasImpactOnPolicy := 
+	hasImpactOnPolicy :=
 		(util.InSlice(columns, "groups") && !slices.Equal(oldUser.Groups, user.Groups)) ||
-		(util.InSlice(columns, "name") && oldUser.Name != user.Name) ||
-		(util.InSlice(columns, "owner") && oldUser.Owner != user.Owner)
+			(util.InSlice(columns, "name") && oldUser.Name != user.Name) ||
+			(util.InSlice(columns, "owner") && oldUser.Owner != user.Owner)
```
Comment (dplynskiy) [inline object/user.go:609]: redunant spaces

File: object/check.go (hunk 1 above)
Comment (Aleksgav) [reply object/check.go:398]: it's just a mapping errors to only needed events and it description

Comment (dplynskiy) [reply object/check.go:398]: but all detail I see here exactly repeats error texts

File: object/check.go [hunk 3]
```diff
@@ -283,7 +284,58 @@ func checkLdapUserPassword(user *User, password string, lang string) string {
 	return ""
 }
 
-func CheckUserPassword(organization string, username string, password string, lang string, options ...bool) (*User, string) {
+var ErrorUserNotFound = errors.New("user not found")
+var ErrorUserDeleted = errors.New("user deleted")
+var ErrorUserBlocked = errors.New("user deleted")
```
Comment (dplynskiy) [inline object/check.go:289]: text should be user blocked

Comment (Aleksgav) [reply object/check.go]: fixed

**Review by dplynskiy — APPROVED** [review]

File: object/check.go (hunk 1 above)
Comment (Aleksgav) [reply object/check.go]: fixed

**Pull request merged**